src/config/mod.rs

```diff
@@ -3499,6 +3501,7 @@ impl Default for TikvConfig {
             abort_on_panic: false,
             memory_usage_limit: None,
             memory_usage_high_water: 0.9,
+            hybrid_engine_enabled: false,
```

Comment:
The name is too hard for user to understand. I think we may just use the region-cache-memory-quota to enable or disable the feature. By default, it's empty which means it's disabled.

Reply:
Good suggestion.